ocaml/xapi/importexport.ml

```diff
@@ -277,7 +277,13 @@ let remote_metadata_export_import ~__context ~rpc ~session_id ~remote_address ~r
                         )
                      )
                 )
-             );
+             )
+           with e ->
+             begin match e with
```

Comment:
You should not need `begin/end` here. I would rather put them around the `try` block in place of the additional pair of brackets.